kmir/src/tests/integration/data/exec-smir/call-with-args/main-a-b-with-int.state

```diff
@@ -1,6 +1,6 @@
 <kmir>
   <k>
-    #execBlock ( basicBlock (... statements: .Statements , terminator: terminator (... kind: terminatorKindReturn , span: span ( 63 ) ) ) ) ~> .K
+    Integer ( 11 , 16 , true ) ~> #freezer#setLocalValue(_,_)_RT-DATA_KItem_Place_Evaluation1_ ( place (... local: local ( 2 ) , projection: .ProjectionElems ) ~> .K ) ~> #setArgsFromStack ( 3 , .Operands ) ~> #execBlock ( basicBlock (... statements: .Statements , terminator: terminator (... kind: terminatorKindReturn , span: span ( 63 ) ) ) ) ~> .K
```

Comment:
The idea here was to execute exactly  to the point where the (empty) function body is executed, so we would want to adjust the step count in `test_integration.py` if we were to keep that idea.